frontend/src/Dashboard/CrossingDetailPage/CrossingDetails.js

```diff
@@ -1,12 +1,27 @@
 import React, { Component } from 'react';
 
 class CrossingDetails extends Component {
+
   render() {
+    const { crossing, community } = this.props;
+
     return (
-      <div>Crossing Details: {this.props.crossingId}</div>
+      <div>
+        <div>Crossing Details: {crossing.id}</div>
```

Comment:
minor, I don't believe we're passing id back with the crossing.

Reply:
just noticed that this wasn't fixed in this PR, it's fixed in the edit crossings pr